tests/fast/SwizzledRollbackSideband.txt

```diff
@@ -29,4 +29,6 @@ testTitle=SwizzledCausalConsistencyTest
     minDelay=0
     maxDelay=100
     kill1Timeout=30
-    kill2Timeout=6000
\ No newline at end of file
+    kill2Timeout=6000
+
+minimumReplication=2
```

Comment:
I'm hesitate to add this constraint because it will affect other test scenarios when 1 replica is legit. 

I assume you want to avoid the false positive when the test workload `FORCEly` marks a SS as failed. 
I'm thinking we may test the `FORCEly` feature by only marking one server as failed when there are more than 1 SS to kill.  Since it is legit to kill more than 1 SS in that arbitrary setting, marking 1 SS without checking the safety should not destroy the DB.

Reply:
To be clear, is the request to only remove the constraint from this workload or all of them? I have no complaints against making the changes and having it run through testing to see how it is handled.

Reply:
Just remove the constraint and see what happens in simulation.

Reply:
Removing this will not work in conjunction with the idea that you suggested using `DD_EXCLUDE_MIN_REPLICAS` knob setting the minimum number of replicas as 2 under buggify. We will have to compromise on one or the other here I believe.

Reply:
OK. I prefer to not change the test case. 
You can drop `DD_EXCLUDE_MIN_REPLICAS ` suggestion.

Reply:
I kept the knob but removed the buggify setting